pkg/setting/setting.go

```diff
@@ -1459,7 +1460,8 @@ func readAuthSettings(iniFile *ini.File, cfg *Cfg) (err error) {
 
 	// Debug setting unlocking frontend auth sync lock. Users will still be reset on their next login.
 	cfg.DisableSyncLock = auth.Key("disable_sync_lock").MustBool(false)
-
+	// Do not use
+	cfg.AuthConfigUIServerAdminOnly = auth.Key("config_ui_server_admin_only").MustBool(true)
```

Comment:
```suggestion
	cfg.AuthConfigUIAdminAccess = auth.Key("config_ui_admin_access").MustBool(false)
```

just testing to get a sense if this is more appropriate, to be more understanding of what the configuration actually does. even if it's only internal

Reply:
I like this, I think it's a bit more readable.